Packs/Salesforce/Integrations/SalesforceV2/SalesforceV2.py

```diff
@@ -0,0 +1,1041 @@
+import json
+import traceback
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import dateparser
+#import demistomock as demisto  # noqa: F401
+import urllib3
+#from CommonServerPython import *  # noqa: F401
+from dateparser import parse
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+MAX_INCIDENTS_TO_FETCH = 50
```

Comment:
max_fetch should be a parameter of the integration, allowing the user to choose maximum fetched incidents per fetch.

Reply:
This comment is not relevant anymore because the Pack's support level has changed to - 'community'.